viewer/packages/cad-parsers/src/cad/V9GeometryFilterer.test.ts

```diff
@@ -0,0 +1,95 @@
+/*!
+ * Copyright 2021 Cognite AS
+ */
+
+import { V9GeometryFilterer } from './V9GeometryFilterer';
+import { RevealGeometryCollectionType } from '@reveal/sector-parser';
+import * as THREE from 'three';
+
+import {
+  PrimitiveType,
+  Ellipsoid,
+  Box,
+  getCollectionType
+} from '../../../../test-utilities/src/primitives/primitiveTypes';
+
+import {
+  createPrimitiveInterleavedGeometry,
+  parseInterleavedGeometry
+} from '../../../../test-utilities/src/primitives/primitiveThreeBuffers';
+
+function assertSecondCulledAwayFromBoxAtX10Yneg10Z0(primitives: any[], primitiveType: PrimitiveType) {
```

Comment:
This function doesn't only assert - it also performs the test. Either rename the function or split into a function that performs the filtering and one that does the assert.